package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-dimensions",
-  "version": "2.0.0-alpha1",
+  "name": "@zillow/react-dimensions",
```

Comment:
I would rather these changes were in a separate PR (or just commit them directly to master, I assume we're not going to push anything upstream anymore?)

Reply:
(the name and version of a fork don't _need_ to be versioned, of course)

Reply:
master has been abandoned -- perhaps we can take ownership of it at some point